strings/en/global.json

```diff
@@ -29,35 +29,35 @@
 		"wrongLink": "Wrong link!"
 	},
 	"roles": {
-		"noLevel": "You must be at least chat level %%level%% to receive this role! You can check your current chat level using %%command%% in %%channel%%.",
-		"successGive": "Successfully gave you the %%role%% role!",
-		"successTake": "Successfully took the %%role%% role from you!"
+		"noLevel": "You must be at least chat level {level} to receive this role! You can check your current chat level using {command} in {channel}.",
```

Comment:
yeah uhm can we not do this please

Reply:
wdym?

Reply:
I'm not gonna merge this with every single variable changed

Reply:
You have access to the branch. You can do as you need.

Reply:
But it seems like you changed the code and I'm not yet familiar with this package so if possible I'd like you to keep the variables that don't need to interact with this package to be kept as they were and for that check to be added in the code